src/commands/feedback.js

```diff
@@ -1,6 +1,57 @@
 const { SlashCommandBuilder } = require('@discordjs/builders');
 const { MessageActionRow, MessageButton, MessageEmbed } = require('discord.js');
 
+async function collector(interaction, message, embed) {
+
+	const filterReaction = (reaction, user) => {
+		return !user.bot && (reaction.emoji.name === '🟩' || reaction.emoji.name === '🟨' || reaction.emoji.name === '🟥');
```

Comment:
```suggestion
		return !user.bot;
```
Unecessary, since collect event handles only these three emojis.